ko.json

```diff
@@ -0,0 +1,296 @@
+            "line2" : "어제의 아이템 상점은 [여기](/shop/yesterday|tooltip=어제의 상점)에서 볼 수 있습니다.",
+            "line3" : "아이템에 대한 자세한 내용을 보려면 해당 아이템을 클릭해보세요.",
+            "countdown" : {
+                "title" : "아이템 변경:",
+                "wordyFormat" : "[number] [noun]",
+                "hour-singular" : "시간",
+                "hours-plural" : "시간"
+            },
+            "yesterday" : {
+                "title" : "어제의 상점"
+            },
+            "vote" : {
+                "title" : "오늘의 상점은 어떤가요?",
+                "line1" : "이 투표는 00:00 UTC에 초기화됩니다.",
+                "line2" : "투표하기 전 [개인정보처리방침](/privacy)을 확인하세요.",
+                "choice" : {
+                    "up" : "좋아요!",
+                    "neutral" : "그저 그래요.",
+                    "down" : "싫어요."
+                },
+                "response" : {
+                    "success" : "투표 기록",
+                    "timeout" : "요청에 대해 응답하는데 너무 오래 걸립니다.",
+                    "internal" : "내부 오류",
+                    "failover" : "[server]에 요청하는데 실패, 다시 시도하는 중...",
+                    "ratelimit" : "요청이 너무 많습니다. 잠시 후에 다시 시도하세요.",
+                    "unavailable" : "투표는 상점이 바뀌는 몇분 뒤까지 이용할 수 없습니다.",
+                    "conflict" : "투표할 수 없습니다. 잠시 후에 다시 시도하세요.",
+                    "bad-choice" : "투표를 인식하지 못함",
+                    "no-data" : "데이터를 이용할 수 없습니다 - 잠시 후에 다시 시도하세요."
+                }
+            }
+        },
+        "upcoming" : {
+            "title" : "곧 출시하는 아이템",
+            "line1" : "포트나이트 배틀로얄의 미발매 아이템"
+        },
+        "list" : {
+            "title" : "치장 아이템 목록",
+            "line1" : "사용할 수 있는 포트나이트 배틀로얄의 모든 치장 아이템.",
+            "table" : {
+                "icon" : "아이콘",
+                "name" : "이름",
+                "type" : "종류",
+                "rarity" : "희귀도",
+                "obtained" : "획득 방법"
+            }
+        },
+        "history" : {
+            "list" : {
+                "title" : "아이템 상점 기록",
+                "line1" : "지난 8일간의 포트나이트 배틀로얄 아이템 상점의 기록입니다.",
+                "line2" : "해당일의 모든 아이템 상점 기록을 보려면 클릭하세요."
+            },
+            "view" : {
+                "title" : "일일 상점 목록",
+                "line1" : "[date]의 포트나이트 배틀로얄 상점 목록입니다. ",
+                "line2" : "아이템에 대한 자세한 내용을 보려면 해당 아이템을 클릭해보세요.",
+                "today-button" : "오늘의 아이템 상점"
+            }
+        },
+        "news" : {
+            "title" : "뉴스",
+            "line1" : "에픽게임즈의 인게임 뉴스피드입니다. 매분 업데이트됩니다.",
+            "error" : "포트나이트로부터 뉴스를 가져올 수 없습니다. 곧 다시 시도합니다.",
+            "more-ingame-button" : "더보기"
+        },
+        "randomiser" : {
+            "title" : "랜덤 아이템",
+            "line1" : "사용 가능한 모든 의상 중 무작위로 보여줍니다.",
+            "chooseText" : "클릭하여 선택하세요",
+            "randomise-all" : "모두 섞기"
+        },
+        "png" : {
+            "title" : "치장 아이템 PNG",
+            "line1" : "Please credit back to fnbr.co if you use any of these images. Previews are scaled down, downloads will be full size.",
```

Comment:
Would it be possible to translate this string?

Reply:
End.